packages/manager/modules/pci/src/projects/new/payment/index.html

```diff
@@ -55,181 +55,188 @@ <h1 data-translate="pci_project_new_payment_title"></h1>
-                <a
-                    data-ng-if="$ctrl.needToCheckCustomerInformations"
-                    class="oui-button oui-button_primary oui-button_icon-right text-white"
-                    data-ng-href="{{ $ctrl.orderBillingUrl }}"
-                    data-ng-click="$ctrl.sendTrack('project-creation::track-order')"
-                    target="_blank"
-                    rel="noopener"
-                >
-                    <span
-                        data-translate="pci_project_new_payment_check_anti_fraud_case_fraud_manual_review_link"
-                    ></span>
-                    <span
-                        class="oui-icon oui-icon-arrow-right"
-                        aria-hidden="true"
-                    ></span>
-                </a>
-
-                <!--"Create project" and "continue" btn for Credit and PayPal-->
-                <div data-ng-if="!$ctrl.needToCheckCustomerInformations">
-                    <!--Continue btn for default payment method-->
-                    <div
-                        class="d-flex"
-                        data-ng-if="$ctrl.canProceedToCreateProject()"
-                    >
-                        <oui-button
-                            data-variant="primary"
-                            data-type="submit"
-                            data-variant-nav="next"
-                            data-disabled="$ctrl.isInvalidPaymentMethod()"
-                            data-on-click="$ctrl.sendTrack('new_project_payment_continue')"
-                        >
-                            <span
-                                data-translate="pci_project_new_payment_btn_continue_default"
-                            ></span>
-                        </oui-button>
-                        <oui-spinner
-                            class="m-2"
-                            size="s"
-                            data-ng-if="$ctrl.globalLoading.finalize"
-                        >
-                        </oui-spinner>
-                    </div>
-
-                    <!--Create project btn for Voucher (CREDIT)-->
-                    <div
-                        class="d-flex"
-                        data-ng-if="$ctrl.model.paymentMethod.paymentType === 'CREDIT'"
+                    <!--Credit card and PayPal-->
+                    <ovh-payment-method-integration
+                        data-callback-status-param-url-name="paymentStatus"
+                        data-payment-method-type="$ctrl.model.paymentMethod"
+                        data-on-initialized="$ctrl.onIntegrationInitialized.bind($ctrl)"
+                        data-on-submit="$ctrl.onIntegrationSubmit.bind($ctrl)"
+                        data-on-submit-error="$ctrl.onIntegrationSubmitError.bind($ctrl)"
+                        data-on-submit-success="$ctrl.onIntegrationSubmitSuccess.bind($ctrl)"
+                        data-ng-if="$ctrl.displayCreateProjectButton()"
```

Comment:
I have a question for you for this check in your parent PR.

Reply:
https://github.com/ovh/manager/pull/9847

Reply:
Ok, I'll check it in the parent PR.
Beside this, are you Ok with the review of this PR so I can merge with its parent ?

Reply:
yes good Rafik